dataedit/templates/dataedit/dataview.html

```diff
@@ -155,126 +155,18 @@ <h4 class="panel-title">
-        <tr>
-            <th>Sources</th>
-            <td>
-
-                {% for row in comment_on_table.sources %}
-                    <p class="metaproperty">
-                    {% if row.url %}
-                        <a href="{{row.url}}">{{row.name}}</a>
-                    {% else %}
-                        {{row.name}}
-                    {% endif %}
-                    </p>
-                    <ul>
-                        <li> Description: {{row.description }}</li>
-                        <li> License: {{row.license }}</li>
-                    </ul>
-                {% endfor %}
-            </td>
-        </tr>
-        {% endif %}
-		
-        {% if comment_on_table.license %}
-        <tr>
-            <th>License</th>
-            <td>
-                    <p class="metaproperty">
-                        {% if comment_on_table.license.url %}
-                            <a href="{{comment_on_table.license.url}}">
-                        {% endif %}
-                        {{comment_on_table.license.name}} ({{comment_on_table.license.id}})
-                        {% if comment_on_table.license.url %}
-                            </a>
-                        {% endif %}
-                    </p>
-                    <ul>
-                        <li> Version: {{comment_on_table.license.version }}</li>
-                        <li> Instruction: {{comment_on_table.license.instruction }}</li>
-                        <li> Copyright: {{comment_on_table.license.copyright }}</li>
-                    </ul>
-            </td>
-        </tr>
-        {% endif %}
-        {% if comment_on_table.contributors %}
-        <tr>
-            <th>Contributors</th>
-            <td>
-                {% for row in comment_on_table.contributors %}
-                    <p class="metaproperty">
-                        {% if row.email %}
-                            <a href="mailto:{{row.email}}">{{row.name}} ({{row.date}})</a>
-                        {% else %}
-                            {{row.name}} ({{row.date}})
-                        {% endif %}
-                    </p>
-                    {{row.comment}}
-                {% endfor %}
-            </td>
-        </tr>
-        {% endif %}
-        {% if comment_on_table.fields %}
-        <tr>
-            <th>Columns</th>
-            <td>
-                    {% for source in comment_on_table.fields %}
-                        <p class="metaproperty">{{source.name}} {% if source.unit %} (in {{source.unit}}) {% endif %}</p>
-                        {% if source.description %}
-                            {{source.description}}
-                        {% else %}
-                            No description
-                        {% endif %}
-                        <hr>
-                    {% endfor %}
-            </td>
-        </tr>
-        {% endif %}
-    </table>
+     <h4>Metadata</h4><a href="{{table}}/meta_edit"> Edit</a>
+        {{meta_widget}}
```

Comment:
The render function of the widget automatically generated the code (and can do so for a future updated version of metadata)